src/crafting_gui.cpp

```diff
@@ -625,25 +625,14 @@ int recipe::print_items( WINDOW *w, int ypos, int xpos, nc_color col, int batch
     const int oldy = ypos;
 
     mvwprintz( w, ypos++, xpos, col, _( "Byproducts:" ) );
-    for( auto &bp : byproducts ) {
-        print_item( w, ypos++, xpos, col, bp, batch );
+    for( const auto &bp : byproducts ) {
+        item it( bp.first, calendar::turn, item::default_charges_tag{} );
+        mvwprintz( w, ypos++, xpos, col, _( "> %d %s" ), bp.second * batch, it.tname().c_str() );
```

Comment:
Can `item::display_name()` be used here? (to prevent constructing full blown object above).